templates/jetty.xml.erb

```diff
@@ -31,24 +31,6 @@ DTD Configure//EN" "http://jetty.mortbay.org/configure.dtd">
     <!-- Use this connector for many frequently idle connections and for
         threadless continuations. -->
     <Call name="addConnector">
-        <Arg>
-            <New class="org.eclipse.jetty.server.nio.SelectChannelConnector">
-                <Set name="host"><%= scope.lookupvar('opendaylight::odl_bind_ip') %>
-                    <Property name="jetty.host" />
-                </Set>
-                <Set name="port">
-                    <Property name="jetty.port" default="8181" />
-                </Set>
-                <Set name="maxIdleTime">300000</Set>
-                <Set name="Acceptors">2</Set>
-                <Set name="statsOn">false</Set>
-                <Set name="confidentialPort">8543</Set>
-                <Set name="lowResourcesConnections">20000</Set>
-                <Set name="lowResourcesMaxIdleTime">5000</Set>
-            </New>
-        </Arg>
-    </Call>
-    <Call name="addConnector">
```

Comment:
Yeah, that's exactly what I was seeing. Removed comment because I saw the strange diff